### PR TITLE
ICU-21349 Small improvements to MeasureUnit comments

### PR DESCRIPTION
**Pull request by younies**
Closes: https://github.com/icu-units/icu/issues/77

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

File: icu4c/source/i18n/unicode/measunit.h [hunk 1]
```diff
@@ -335,7 +335,7 @@ class U_I18N_API MeasureUnit: public UObject {
 
 #ifndef U_HIDE_DRAFT_API
     /**
-     * Get the CLDR Unit Identifier for this MeasureUnit, as defined in UTS 35.
+     * Get CLDR Unit Identifier for this MeasureUnit, as defined in UTS 35.
```
Comment (hugovdm) [inline icu4c/source/i18n/unicode/measunit.h:338]: Why this change?

**Comment by hugovdm** [comment]
Please make the PR description clearer too: it helps git history to be useful, maybe something like "ICU-21349 Small javadoc fix in MeasureUnit.java"? squashbot can then rewrite the commit for us.

File: icu4c/source/i18n/unicode/measunit.h (hunk 1 above)
Comment (younies) [reply icu4c/source/i18n/unicode/measunit.h:338]: shall we add `the`?

Comment (hugovdm) [reply icu4c/source/i18n/unicode/measunit.h:338]: I don't mind either way, but how about keeping measunit.h getIdentifier() and MeasureUnit.java's getIdentifier() in sync?

**Review by hugovdm — APPROVED** [review]

File: icu4c/source/i18n/unicode/measunit.h (hunk 1 above)
Comment (hugovdm) [reply icu4c/source/i18n/unicode/measunit.h:338]: Modified, squashed, PR title adjusted. Marking as resolved.
